common/lib/xmodule/xmodule/tabs.py

```diff
@@ -121,14 +133,8 @@ def __setitem__(self, key, value):
 
         Note: the 'type' member can be 'get', but not 'set'.
```

Comment:
I guess if we are trying to determine what level of testing is required, or we we might take shortcuts, this is a good candidate for a shortcut.